src/engine/debugging/PerformanceMetrics.cs

```diff
@@ -41,6 +41,9 @@ private PerformanceMetrics()
         instance = this;
     }
 
+    [Signal]
+    public delegate void OnHidden();
```

Comment:
I just remembered that popups have `popup_hide` signal, so that should be used instead of adding a new one unless there's a reason why a separate is needed.

Reply:
PerformanceMetrics is a Control...

Reply:
Ah yeah I missed that.